src/components/Footer.astro

```diff
@@ -0,0 +1,47 @@
+---
+// Please mention any ideas for links as such ...
+---
+
+<footer class="bg-dark-blue mt-3 hover:shadow">
+    <div class="mx-auto w-full max-w-screen-xl p-4 py-6 lg:py-8">
+        <div class="md:flex md:justify-between">
+          <div class="mb-6 md:mb-0">
+              <a href="https://lfx.linuxfoundation.org/tools/mentorship/" class="flex items-center" target="_blank">
+                  <span class="self-center text-2xl font-semibold whitespace-nowrap text-white">LFX Mentorship</span>
+              </a>
+          </div>
+      </div>
+      <hr class="my-6 sm:mx-auto border-gray-500 lg:my-8" />
+      <div class="flex gap-2 items-end justify-between md:items-center md:justify-between">
+          <span class="text-sm text-gray-500 text-center dark:text-gray-400">
+            <a href="https://www.copsiitbhu.co.in/" class="hover:underline" target="_blank">COPS @IIT(BHU)-Varanasi</a>
+          </span>
+          <div class="flex mt-4 space-x-5 justify-center sm:mt-0">
+              <a href="#" class="text-gray-500 hover:text-blue-600 duration-200">
+                  <svg class="w-5 h-5" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 8 19">
+                        <path fill-rule="evenodd" d="M6.135 3H8V0H6.135a4.147 4.147 0 0 0-4.142 4.142V6H0v3h2v9.938h3V9h2.021l.592-3H5V3.591A.6.6 0 0 1 5.592 3h.543Z" clip-rule="evenodd"/>
+                    </svg>
+                  <span class="sr-only">Facebook page</span>
+              </a>
+              <a href="#" class="text-gray-500 hover:text-purple-900 duration-200">
+                  <svg class="w-5 h-5" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 21 16">
```

Comment:
Instead of adding SVGs directly, store them separately in `assets` directly in `src` and then import them into the component.